gradle/libs.versions.toml

```diff
@@ -37,6 +40,7 @@ slf4j-api = { module = "org.slf4j:slf4j-api", version = "2.0.5" }
 apache-log4j-slf4jImpl = { module = "org.apache.logging.log4j:log4j-slf4j2-impl", version = "2.20.0" }
 apache-commons-lang3 = { module = "org.apache.commons:commons-lang3", version = "3.12.0" }
 apache-kafka-kafkaStreamsTestUtils = { module = "org.apache.kafka:kafka-streams-test-utils", version = "7.2.1-ccs" }
+kafka-protobufserde = { module = "io.confluent:kafka-streams-protobuf-serde", version.ref = "confluent-platform" }
```

Comment:
kafka versions should be coming from the kafka-bom, should be able to remove the declaration here and in the one above.